packages/evolution-backend/src/api/survey.user.routes.ts

```diff
@@ -211,5 +211,19 @@ export default (authorizationMiddleware, loggingMiddleware: InterviewLoggingMidd
         }
     );
 
+    router.post('/survey/clientSideException/', async (req: Request, res: Response) => {
+        try {
+            const content = req.body;
+            const interviewId = content.interviewId || -1;
+            if (content.exception) {
+                console.error('Client-side exception in interview %d: %s', interviewId, content.exception);
```

Comment:
is console.error safe to log any kind of content being sent from a client? Do we need to cap the length or other security protections ?